### ISSUE TITLE
Keyboard focus order is not logical on the tile cards using tab key in windows.

### ISSUE THREAD

**Issue by msftedad** [bvaughn/react-virtualized#1889]
**Test Environment:**
OS: Windows 11 version 25H2 (OS Build 26200.7271)
Browser: Edge (Version 144.0.3719.18)
URL: https://codepen.io/Sravani-Reddy-the-vuer/pen/xbVojEZ

**User Impact:**
Keyboard‑only users may get confused when focus jumps on an illogical order, making it hard to follow the layout. This can cause missed information and increase effort while navigating tile cards.

**Repro Steps:**

1. Open the URL: https://codepen.io/Sravani-Reddy-the-vuer/pen/xbVojEZ in edge browser in windows.
2. Press tab key to navigate to the Cards.
3. Verify whether keyboard focus order is logical on the tile cards using tab key in windows or not.

**Actual Result:**
Keyboard focus order is not logical on the tile cards using tab key in windows.
Current focus order: Card 5 > Card 4 > Card 3 > Card 2> Card 1

**Refer Attachment:**

1. Keyboard focus order is not logical on the tile cards using tab key in windows.png
2. Keyboard focus order is not logical on the tile cards using tab key in windows.mp4

https://github.com/user-attachments/assets/33bea399-09c7-4881-a780-0e8abd8b4394
<img width="969" height="520" alt="Image" src="https://github.com/user-attachments/assets/7a558de5-8727-4e6d-aa12-60b669d25ffa" />

**Expected Result:**
Keyboard focus order should be logical on the tile cards using tab key in windows.
Expected Focus order: Card 1 > Card 2 > Card 3 > Card 4> Card 5
